tjreports/site/models/reports.php

```diff
@@ -177,9 +177,8 @@ protected function setCustomFieldsColumns()
 		{
 			$customField = array (
 				'title'              => ($columnLabels[$columnName]) ? $columnLabels[$columnName] : $columnName,
-				'table_column'       => $this->customFieldsTableAlias . '.' . $columnName
-
-				// , 'disable_sorting' => true
+				'table_column'       => $this->customFieldsTableAlias . '.' . $columnName,
+				'not_show_hide'      => false
```

Comment:
will this also hide actual custom fields created using com_fields by default? @vaibhavsTekdi